### ISSUE TITLE
add support for Gnome 47

### ISSUE THREAD

**Issue by thekswenson** [konkor/cpufreq#227]
The extension doesn't currently work with Gnome 47 (ubuntu 24.10).

I noticed that @DisDis has made changes for Gnome 46: https://github.com/DisDis/cpufreq/pull/1
I will try to clone these changes and update the `metadat.json` files. With luck, this will be enough to run cpufreq in GS47.

**Comment by thekswenson** [comment]
I've cloned this repo: https://github.com/gilvbp/cpufreq/tree/gnome_46 (and switched to the gnome_46 branch)
After running `autogen.sh` I change the version numbers in `metadata.json`.  Then I can enable the extension in GS47.

There is a problem, however, in that the `cpufreqctl` commands use 100% cpu for some reason.  I have to disable the extension and then kill the many `cpufreqctl` commands spawned by the process.

The strange thing is that, if I call the command myself:
```
/usr/bin/pkexec /usr/bin/cpufreqctl --governor --core=3 --set performance
```
there are no issues, and the command terminates normally.

**Comment by cogitech2** [comment]
Yes, I am sure there are a lot of people who would love to have a version of this extension for Gnome 46/47, including me. I wish I could help...

**Comment by GaVenga** [comment]
The extension doesn't currently work with Gnome 48.
Seems to be dead...
